src/util/scroll.js

```diff
@@ -1,7 +1,7 @@
 import scrollToIndex from './scrollToIndex.js';
 import { getToolState } from '../stateManagement/toolState.js';
 
-export default function (element, images) {
+export default function (element, images, loop) {
```

Comment:
Can you set a default value? (false, I guess.

Reply:
done :)